samples/bluetooth/path_loss_monitoring/central/sysbuild/ipc_radio/prj.conf

```
@@ -0,0 +1,7 @@
+CONFIG_BT_CTLR=y
+CONFIG_BT_LL_SOFTDEVICE=y
+CONFIG_BT=y
+CONFIG_BT_PATH_LOSS_MONITORING=y
+
+CONFIG_IPC_RADIO_BT=y
+CONFIG_IPC_RADIO_BT_HCI_IPC=y
```

```suggestion
CONFIG_BT_PATH_LOSS_MONITORING=y
```
Should only require this if we do the Kconfig.sysbuild change